packages/react/src/components/Toolbar/ToolbarCustomItem.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+
+import {
+  ChildrenComponentProps,
+  ContentComponentProps,
+  createShorthandFactory,
+  UIComponentProps,
+  UIComponent,
+  childrenExist,
+  commonPropTypes,
+} from '../../lib'
+
+import { WithAsProp, withSafeTypeForAs } from '../../types'
+import { Accessibility } from '../../lib/accessibility/types'
+import { defaultBehavior } from '../../lib/accessibility'
+
+export interface ToolbarCustomItemProps
+  extends UIComponentProps,
```

Comment:
is it possible to restrict the prop types and remove onClick? onClick should be put on the content element instead as the custom item is not focusable.

Reply:
probably, I am missing something - but currently not sure how aforementioned statement can coexist with `focusable` prop :)

```jsx
  /** A custom item can be focused. */
  focusable?: boolean
```

@miroslavstastny, @jurokapsiar, could you, please, suggest which part I am missing?

Reply:
Per offline discussion: it should be `focusable` conditionally, but it can't be `actionable` (`onClick: never`). We can consider to change this later based on provided use cases.